src/api/app_updated_section.py

```
@@ -0,0 +1,8 @@
+# Include instance management routes
+app.include_router(instances_router, prefix="/api/v1", tags=["instances"])
```

Ruff (0.12.2)

Undefined name `app`

(F821)